Floofbot/Services/EventHandlerService.cs

```diff
@@ -114,6 +115,28 @@ private async Task CheckUserAutoban(IGuildUser user)
                 }
             }
         }
+
+        // rfurry rules gate
+        public async Task RulesGate(SocketMessage msg)
+        {
+            var userMsg = msg as SocketUserMessage;
+            if (msg == null || msg.Author.IsBot)
+                return;
+
+            // rules gate info
+            ulong rulesChannelId = 225980129799700481;
```

Comment:
All 4 of these values should be in the config

(See below, you might be able to get rid of `rulesServerId`)

Reply:
I was unsure about adding it to the config, as its a pretty specific thing for a specific server

Reply:
It's a good practice thing. Not having stuff like this configurable has bitten us in the collective ass previously, and I'd like to avoid something like that happening even if it seems very unlikely (which it is).